Klee: Version 1.100

aaronbell opened the pull request:
fcd7dec: [gftools-packager] Klee: Version 1.000; ttfautohint (v1.8.3) added

* Klee Version 1.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontworks-fonts/Klee.git at commit https://github.com/fontworks-fonts/Klee/commit/7b57d36ecd1d715e0a1c0fc7d74b1f871a2ce400.

 edcf471: [gftools-packager] ofl/klee remove METADATA "source".  google/fonts#2587

RosaWagner commented:
There is a conflict between Metadata.pb and the name table that is creating a lot of Fails (Klee vs Klee One).

aaronbell commented:
Updated

## Klee One: Version 1.000; ttfautohint (v1.8.3) added

---
 af42209: [gftools-packager] Klee One: Version 1.000; ttfautohint (v1.8.3) added

* Klee One Version 1.000; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/fontworks-fonts/Klee.git at commit https://github.com/fontworks-fonts/Klee/commit/7b57d36ecd1d715e0a1c0fc7d74b1f871a2ce400.

 702b91f: [gftools-packager] ofl/kleeone remove METADATA "source".  google/fonts#2587

aaronbell commented:
Ah! Thanks. The repro got replaced and the upstream file wasn't updated.

RosaWagner commented:
## Fontbakery report

Fontbakery version: 0.7.33

<details>
<summary><b>[15] KleeOne-Regular.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> METADATA.pb font.name value should be same as the family name declared on the name table.</summary>

* [com.google.fonts/check/metadata/nameid/font_name](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/nameid/font_name)

* 🔥 **FAIL** Unmatched familyname in font: TTF has familyname = "クレー One" while METADATA.pb has font.name = "Klee One". [code: mismatch]

</details>
<details>
<summary>🔥 <b>FAIL:</b> METADATA.pb font.name field contains font name in right format?</summary>

* [com.google.fonts/check/metadata/valid_name_values](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/valid_name_values)

* 🔥 **FAIL** METADATA.pb font.name field ("Klee One") does not match correct font name format ("クレー One"). [code: mismatch]

</details>
<details>
<summary>🔥 <b>FAIL:</b> METADATA.pb font.full_name field contains font name in right format?</summary>

* [com.google.fonts/check/metadata/valid_full_name_values](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/valid_full_name_values)

* 🔥 **FAIL** METADATA.pb font.full_name field ("Klee One Regular") does not match correct font name format ("クレー One"). [code: mismatch]

</details>
<details>
<summary>🔥 <b>FAIL:</b> METADATA.pb font.post_script_name field contains font name in right format?</summary>

* [com.google.fonts/check/metadata/valid_post_script_name_values](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/valid_post_script_name_values)

* 🔥 **FAIL** METADATA.pb postScriptName ("KleeOne-Regular") does not match correct font name format ("クレー One"). [code: mismatch]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries.</summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
<pre>--- Rationale ---

Checks that the family name infered from the font filename matches the string
at nameID 1 (NAMEID_FONT_FAMILY_NAME) if it conforms to RIBBI and otherwise
checks that nameID 1 is the family name + the style name.


</pre>

* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):WINDOWS(3)] on the "name" table: Expected "Klee One" but got "クレー One". [code: mismatch]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API?</summary>

* [com.google.fonts/check/metadata/listed_on_gfonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts)

* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check if each glyph has the recommended amount of contours.</summary>

* [com.google.fonts/check/contour_count](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/contour_count)
<pre>--- Rationale ---

Visually QAing thousands of glyphs by hand is tiring. Most glyphs can only be
constructured in a handful of ways. This means a glyph&#x27;s contour count will
only differ slightly amongst different fonts, e.g a &#x27;g&#x27; could either be 2 or 3
contours, depending on whether its double story or single story.

However, a quotedbl should have 2 contours, unless the font belongs to a
display family.

This check currently does not cover variable fonts because there&#x27;s plenty of
alternative ways of constructing glyphs with multiple outlines for each feature
in a VarFont. The expected contour count data for this check is currently
optimized for the typical construction of glyphs in static fonts.


</pre>

* ⚠ **WARN** This check inspects the glyph outlines and detects the total number of contours in each of them. The expected values are infered from the typical ammounts of contours observed in a large collection of reference font families. The divergences listed below may simply indicate a significantly different design on some of your glyphs. On the other hand, some of these may flag actual bugs in the font such as glyphs mapped to an incorrect codepoint. Please consider reviewing the design and codepoint assignment of these to make sure they are correct.

The following glyphs do not have the recommended number of contours:

Glyph name: uni2158	Contours detected: 3	Expected: 4
Glyph name: uni255E	Contours detected: 1	Expected: 2
Glyph name: uni2561	Contours detected: 1	Expected: 2
Glyph name: uni25CC	Contours detected: 24	Expected: 16 or 12
Glyph name: uni25CC	Contours detected: 24	Expected: 16 or 12 [code: contour-count]

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
<pre>--- Rationale ---

All ligatures in a font must have corresponding caret (text cursor) positions
defined in the GDEF table, otherwhise, users may experience issues with caret
rendering.

If using GlyphsApp, ligature carets can be set directly on canvas by accessing
the `Glyph -&gt; Set Anchors` menu option or by pressing the `Cmd+U` keyboard
shortcut.

If designing with UFOs, (as of Oct 2020) ligature carets are not yet compiled
by ufo2ft, and therefore will not build via FontMake. See
googlefonts/ufo2ft/issues/329


</pre>

* ⚠ **WARN** This font lacks caret position values for ligature glyphs on its GDEF table. [code: lacks-caret-pos]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
<pre>--- Rationale ---

Fonts with ligatures should have kerning on the corresponding non-ligated
sequences for text where ligatures aren&#x27;t used (eg
https://github.com/impallari/Raleway/issues/14).


</pre>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l
	- ae + acutecomb
	- acutecomb + grave
	- uni0251 + acutecomb
	- uni0254 + acutecomb
	- uni0259 + acutecomb
	- uni025A + acutecomb
	- uni025B + acutecomb
	- uni028C + acutecomb
	- f.italic + f.italic
	- f.italic + i.italic
	- i.italic + f.italic
	- f.italic + l.italic
	- l.italic + f.italic
	- i.italic + l.italic

   [code: lacks-kern-info]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Designer is listed with the correct name on the Google Fonts catalog of designers?</summary>

* [com.google.fonts/check/metadata/designer_profiles](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/designer_profiles)

* ⚠ **WARN** It seems that Fontworks Inc. is still not listed on the designers catalog. Please submit a photo and a link to a webpage where people can learn more about the work of this designer/typefoundry. [code: profile-not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Glyph names are all valid?</summary>

* [com.google.fonts/check/valid_glyphnames](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/valid_glyphnames)
<pre>--- Rationale ---

Microsoft&#x27;s recommendations for OpenType Fonts states the following:

&#x27;NOTE: The PostScript glyph name must be no longer than 31 characters, include
only uppercase or lowercase English letters, European digits, the period or the
underscore, i.e. from the set [A-Za-z0-9_.] and should start with a letter,
except the special glyph name &quot;.notdef&quot; which starts with a period.&#x27;

https://docs.microsoft.com/en-us/typography/opentype/spec/recom#post-table


In practice, though, particularly in modern environments, glyph names can be as
long as 63 characters.
According to the &quot;Adobe Glyph List Specification&quot; available at:

https://github.com/adobe-type-tools/agl-specification


</pre>

* ⚠ **WARN** The following glyph names may be too long for some legacy systems which may expect a maximum 31-char length limit:
quotedblsharprightalt.half.rotat [code: legacy-long-names]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check mark characters are in GDEF mark glyph class</summary>

* [com.google.fonts/check/gdef_mark_chars](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/gdef.html#com.google.fonts/check/gdef_mark_chars)
<pre>--- Rationale ---

Mark characters should be in the GDEF mark glyph class.


</pre>

* ⚠ **WARN** The following mark characters could be in the GDEF mark glyph class:
	 U+0300, U+0302, U+0303, U+0304, U+030A, U+0327, U+0332, U+0336 and U+20DE [code: mark-chars]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check GDEF mark glyph class doesn't have characters that are not marks)</summary>

* [com.google.fonts/check/gdef_non_mark_chars](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/gdef.html#com.google.fonts/check/gdef_non_mark_chars)
<pre>--- Rationale ---

Glyphs in the GDEF mark glyph class become non-spacing and may be repositioned
if they have mark anchors.
Only combining mark glyphs should be in that class. Any non-mark glyph must not
be in that class, in particular spacing glyphs.


</pre>

* ⚠ **WARN** The following non-mark characters should not be in the GDEF mark glyph class:
	 U+02CA and U+203E [code: non-mark-chars]

</details>
<details>
<summary>⚠ <b>WARN:</b> Do outlines contain any jaggy segments?</summary>

* [com.google.fonts/check/outline_jaggy_segments](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_jaggy_segments)
<pre>--- Rationale ---

This test heuristically detects outline segments which form a particularly
small angle, indicative of an outline error. This may cause false positives in
cases such as extreme ink traps, so should be regarded as advisory and backed
up by manual inspection.


</pre>

* ⚠ **WARN** The following glyphs have jaggy segments:
	* W.half.rotat: B<<511.5,510.0>-<376.0,492.0>-<235.0,476.0>>/B<<235.0,476.0>-<377.0,463.0>-<502.0,448.5>> = 11.704763144225494
	* W.half.rotat: B<<529.0,733.5>-<398.0,719.0>-<235.0,705.0>>/B<<235.0,705.0>-<376.0,692.0>-<509.0,676.0>> = 10.176759266699715
	* W.half: B<<106.5,369.0>-<121.0,238.0>-<135.0,75.0>>/B<<135.0,75.0>-<149.0,216.0>-<164.5,349.0>> = 10.57941948254934
	* W.half: B<<330.0,351.5>-<348.0,216.0>-<364.0,75.0>>/B<<364.0,75.0>-<378.0,217.0>-<392.0,342.0>> = 12.10463872630756
	* beta: B<<632.0,420.0>-<598.0,395.0>-<553.0,386.0>>/B<<553.0,386.0>-<609.0,385.0>-<647.0,359.0>> = 12.33296266268799
	* eight.half.rotat: B<<655.5,514.0>-<692.0,468.0>-<729.0,430.0>>/B<<729.0,430.0>-<727.0,433.0>-<727.0,439.0>> = 10.546034013090223
	* eight.half.rotat: B<<762.0,408.0>-<753.0,408.0>-<744.0,414.0>>/L<<744.0,414.0>--<749.0,409.0>> = 11.309932474020195
	* eight.half: B<<326.5,495.5>-<373.0,532.0>-<410.0,569.0>>/B<<410.0,569.0>-<407.0,567.0>-<401.0,567.0>> = 11.309932474020195
	* eight.half: B<<432.0,602.0>-<432.0,593.0>-<426.0,584.0>>/L<<426.0,584.0>--<431.0,589.0>> = 11.309932474020195
	* eight.italic.rotat: B<<585.0,511.0>-<656.0,382.0>-<729.0,289.0>>/B<<729.0,289.0>-<728.0,291.0>-<727.5,293.0>> = 11.56495651470776 and 390 more. [code: found-jaggy-segments]

</details>
<details>
<summary>⚠ <b>WARN:</b> Do outlines contain any semi-vertical or semi-horizontal lines?</summary>

* [com.google.fonts/check/outline_semi_vertical](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_semi_vertical)
<pre>--- Rationale ---

This test detects line segments which are nearly, but not quite, exactly
horizontal or vertical. Sometimes such lines are created by design, but often
they are indicative of a design error.

This test is disabled for italic styles, which often contain nearly-upright
lines.


</pre>

* ⚠ **WARN** The following glyphs have semi-vertical/semi-horizontal lines:
	* ato.vert: L<<233.0,-14.0>--<234.0,309.0>>
	* ato: L<<683.0,-20.0>--<684.0,303.0>>
	* dorakuma.vert: L<<680.0,430.0>--<681.0,753.0>>
	* dorakuma: L<<221.0,430.0>--<222.0,753.0>>
	* exclam.half.rotat: L<<387.0,617.0>--<774.0,619.0>>
	* exclam.half: L<<223.0,227.0>--<221.0,614.0>>
	* exclam.rotat: L<<387.0,687.0>--<774.0,689.0>>
	* exclam: L<<153.0,227.0>--<151.0,614.0>>
	* exclamdbl: L<<318.0,227.0>--<316.0,614.0>>
	* exclamdbl: L<<649.0,227.0>--<647.0,614.0>> and 18460 more. [code: found-semi-vertical]

</details>
<br>
</details>
<details>
<summary><b>[12] KleeOne-SemiBold.ttf</b></summary>
<details>
<summary>🔥 <b>FAIL:</b> METADATA.pb font.post_script_name field contains font name in right format?</summary>

* [com.google.fonts/check/metadata/valid_post_script_name_values](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/valid_post_script_name_values)

* 🔥 **FAIL** METADATA.pb postScriptName ("KleeOne-SemiBold") does not match correct font name format ("クレー One"). [code: mismatch]

</details>
<details>
<summary>🔥 <b>FAIL:</b> Check name table: FONT_FAMILY_NAME entries.</summary>

* [com.google.fonts/check/name/familyname](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/familyname)
<pre>--- Rationale ---

Checks that the family name infered from the font filename matches the string
at nameID 1 (NAMEID_FONT_FAMILY_NAME) if it conforms to RIBBI and otherwise
checks that nameID 1 is the family name + the style name.


</pre>

* 🔥 **FAIL** Entry [FONT_FAMILY_NAME(1):WINDOWS(3)] on the "name" table: Expected "Klee One SemiBold" but got "クレー One". [code: mismatch]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API?</summary>

* [com.google.fonts/check/metadata/listed_on_gfonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts)

* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check if each glyph has the recommended amount of contours.</summary>

* [com.google.fonts/check/contour_count](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/contour_count)
<pre>--- Rationale ---

Visually QAing thousands of glyphs by hand is tiring. Most glyphs can only be
constructured in a handful of ways. This means a glyph&#x27;s contour count will
only differ slightly amongst different fonts, e.g a &#x27;g&#x27; could either be 2 or 3
contours, depending on whether its double story or single story.

However, a quotedbl should have 2 contours, unless the font belongs to a
display family.

This check currently does not cover variable fonts because there&#x27;s plenty of
alternative ways of constructing glyphs with multiple outlines for each feature
in a VarFont. The expected contour count data for this check is currently
optimized for the typical construction of glyphs in static fonts.


</pre>

* ⚠ **WARN** This check inspects the glyph outlines and detects the total number of contours in each of them. The expected values are infered from the typical ammounts of contours observed in a large collection of reference font families. The divergences listed below may simply indicate a significantly different design on some of your glyphs. On the other hand, some of these may flag actual bugs in the font such as glyphs mapped to an incorrect codepoint. Please consider reviewing the design and codepoint assignment of these to make sure they are correct.

The following glyphs do not have the recommended number of contours:

Glyph name: uni2158	Contours detected: 3	Expected: 4
Glyph name: uni255E	Contours detected: 1	Expected: 2
Glyph name: uni2561	Contours detected: 1	Expected: 2
Glyph name: uni25CC	Contours detected: 24	Expected: 16 or 12
Glyph name: uni25CC	Contours detected: 24	Expected: 16 or 12 [code: contour-count]

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
<pre>--- Rationale ---

All ligatures in a font must have corresponding caret (text cursor) positions
defined in the GDEF table, otherwhise, users may experience issues with caret
rendering.

If using GlyphsApp, ligature carets can be set directly on canvas by accessing
the `Glyph -&gt; Set Anchors` menu option or by pressing the `Cmd+U` keyboard
shortcut.

If designing with UFOs, (as of Oct 2020) ligature carets are not yet compiled
by ufo2ft, and therefore will not build via FontMake. See
googlefonts/ufo2ft/issues/329


</pre>

* ⚠ **WARN** This font lacks caret position values for ligature glyphs on its GDEF table. [code: lacks-caret-pos]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
<pre>--- Rationale ---

Fonts with ligatures should have kerning on the corresponding non-ligated
sequences for text where ligatures aren&#x27;t used (eg
https://github.com/impallari/Raleway/issues/14).


</pre>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l
	- ae + acutecomb
	- acutecomb + grave
	- uni0251 + acutecomb
	- uni0254 + acutecomb
	- uni0259 + acutecomb
	- uni025A + acutecomb
	- uni025B + acutecomb
	- uni028C + acutecomb
	- f.italic + f.italic
	- f.italic + i.italic
	- i.italic + f.italic
	- f.italic + l.italic
	- l.italic + f.italic
	- i.italic + l.italic

   [code: lacks-kern-info]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Designer is listed with the correct name on the Google Fonts catalog of designers?</summary>

* [com.google.fonts/check/metadata/designer_profiles](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/designer_profiles)

* ⚠ **WARN** It seems that Fontworks Inc. is still not listed on the designers catalog. Please submit a photo and a link to a webpage where people can learn more about the work of this designer/typefoundry. [code: profile-not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Glyph names are all valid?</summary>

* [com.google.fonts/check/valid_glyphnames](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/valid_glyphnames)
<pre>--- Rationale ---

Microsoft&#x27;s recommendations for OpenType Fonts states the following:

&#x27;NOTE: The PostScript glyph name must be no longer than 31 characters, include
only uppercase or lowercase English letters, European digits, the period or the
underscore, i.e. from the set [A-Za-z0-9_.] and should start with a letter,
except the special glyph name &quot;.notdef&quot; which starts with a period.&#x27;

https://docs.microsoft.com/en-us/typography/opentype/spec/recom#post-table


In practice, though, particularly in modern environments, glyph names can be as
long as 63 characters.
According to the &quot;Adobe Glyph List Specification&quot; available at:

https://github.com/adobe-type-tools/agl-specification


</pre>

* ⚠ **WARN** The following glyph names may be too long for some legacy systems which may expect a maximum 31-char length limit:
quotedblsharprightalt.half.rotat [code: legacy-long-names]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check mark characters are in GDEF mark glyph class</summary>

* [com.google.fonts/check/gdef_mark_chars](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/gdef.html#com.google.fonts/check/gdef_mark_chars)
<pre>--- Rationale ---

Mark characters should be in the GDEF mark glyph class.


</pre>

* ⚠ **WARN** The following mark characters could be in the GDEF mark glyph class:
	 U+0300, U+0302, U+0303, U+0304, U+030A, U+0327, U+0332, U+0336 and U+20DE [code: mark-chars]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check GDEF mark glyph class doesn't have characters that are not marks)</summary>

* [com.google.fonts/check/gdef_non_mark_chars](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/gdef.html#com.google.fonts/check/gdef_non_mark_chars)
<pre>--- Rationale ---

Glyphs in the GDEF mark glyph class become non-spacing and may be repositioned
if they have mark anchors.
Only combining mark glyphs should be in that class. Any non-mark glyph must not
be in that class, in particular spacing glyphs.


</pre>

* ⚠ **WARN** The following non-mark characters should not be in the GDEF mark glyph class:
	 U+02CA and U+203E [code: non-mark-chars]

</details>
<details>
<summary>⚠ <b>WARN:</b> Do outlines contain any jaggy segments?</summary>

* [com.google.fonts/check/outline_jaggy_segments](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_jaggy_segments)
<pre>--- Rationale ---

This test heuristically detects outline segments which form a particularly
small angle, indicative of an outline error. This may cause false positives in
cases such as extreme ink traps, so should be regarded as advisory and backed
up by manual inspection.


</pre>

* ⚠ **WARN** The following glyphs have jaggy segments:
	* W.half.rotat: B<<457.5,721.5>-<370.0,714.0>-<267.0,705.0>>/B<<267.0,705.0>-<402.0,692.0>-<524.5,678.0>> = 10.494155456025286
	* W.half.rotat: B<<525.5,508.0>-<400.0,492.0>-<264.0,476.0>>/B<<264.0,476.0>-<400.0,463.0>-<515.0,452.0>> = 12.170049399905247
	* W.half: B<<118.5,297.5>-<126.0,210.0>-<135.0,107.0>>/B<<135.0,107.0>-<149.0,242.0>-<162.5,364.5>> = 10.91436469987018
	* W.half: B<<332.5,365.5>-<348.0,240.0>-<364.0,104.0>>/B<<364.0,104.0>-<377.0,240.0>-<388.0,355.0>> = 12.170049399905247
	* eight.box: B<<569.5,482.0>-<613.0,508.0>-<652.0,537.0>>/B<<652.0,537.0>-<650.0,536.0>-<646.0,536.0>> = 10.06906269888942
	* eight.box: B<<673.0,562.0>-<673.0,557.0>-<670.0,552.0>>/B<<670.0,552.0>-<672.0,554.0>-<679.0,559.5>> = 14.036243467926457
	* eight.half.rotat: B<<654.5,513.5>-<690.0,465.0>-<725.0,421.0>>/B<<725.0,421.0>-<724.0,423.0>-<724.0,429.0>> = 11.935602543267397
	* eight.half: B<<326.5,494.5>-<375.0,530.0>-<419.0,565.0>>/B<<419.0,565.0>-<417.0,564.0>-<411.0,564.0>> = 11.935602543267397
	* eight.narrow: B<<575.5,494.5>-<624.0,530.0>-<668.0,565.0>>/B<<668.0,565.0>-<666.0,564.0>-<660.0,564.0>> = 11.935602543267397
	* eight.rotat: B<<655.0,465.0>-<690.0,417.0>-<725.0,373.0>>/B<<725.0,373.0>-<724.0,375.0>-<724.0,381.0>> = 11.935602543267397 and 280 more. [code: found-jaggy-segments]

</details>
<details>
<summary>⚠ <b>WARN:</b> Do outlines contain any semi-vertical or semi-horizontal lines?</summary>

* [com.google.fonts/check/outline_semi_vertical](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_semi_vertical)
<pre>--- Rationale ---

This test detects line segments which are nearly, but not quite, exactly
horizontal or vertical. Sometimes such lines are created by design, but often
they are indicative of a design error.

This test is disabled for italic styles, which often contain nearly-upright
lines.


</pre>

* ⚠ **WARN** The following glyphs have semi-vertical/semi-horizontal lines:
	* T.blackBox: L<<458.0,578.0>--<457.0,169.0>>
	* T.blackBox: L<<546.0,168.0>--<545.0,583.0>>
	* T.half.rotat: L<<267.0,627.0>--<790.0,626.0>>
	* T.half.rotat: L<<795.0,556.0>--<266.0,555.0>>
	* T.half: L<<213.0,107.0>--<214.0,630.0>>
	* T.half: L<<284.0,635.0>--<285.0,106.0>>
	* T.rotat: L<<267.0,589.0>--<792.0,588.0>>
	* T.rotat: L<<796.0,518.0>--<266.0,517.0>>
	* T.roundBox: L<<472.0,169.0>--<473.0,594.0>>
	* T.roundBox: L<<530.0,597.0>--<531.0,168.0>> and 17097 more. [code: found-semi-vertical]

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 7 | 20 | 79 | 15 | 252 | 0 |
| 0% | 2% | 5% | 21% | 4% | 68% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**
* **DEBUG**

davelab6 commented:
This conflicts with "ITC Klee" (https://www.linotype.com/909/klee-family.html) which is indexed in GF (https://fonts.google.com/?query=klee) so we'll need to work with fontwork on a rename

mintoming commented:
> This conflicts with "ITC Klee" (https://www.linotype.com/909/klee-family.html) which is indexed in GF (https://fonts.google.com/?query=klee) so we'll need to work with fontwork on a rename

We agreed to leave it as "Klee" since it's already in the Macintosh system font with the same name.
(Email communication on Jan.20, Dave confirmed)

aaronbell commented:
Thanks Min! As it seems the naming issue has been resolved, should this font be moved back into the queue?

mintoming commented:
Yes I believe we have no problem releasing it :)

aaronbell commented:
Setting this back to "Upstream Resolution" as there is a planned update to the font in the next couple weeks, so it doesn't make sense to push it up now.

aaronbell commented:
Updated

## Klee One: Version 1.100 added

---
 58bdd51: [gftools-packager] Klee One: Version 1.100 added

* Klee One Version 1.100 taken from the upstream repo https://github.com/fontworks-fonts/Klee.git at commit https://github.com/fontworks-fonts/Klee/commit/15a8d8f3e90b3aff71962aefaca4aa83171345e9.

aaronbell commented:
Font updated with improvements from foundry, hinting removed, version pushed to 1.100.

davelab6 commented:
@m4rc1e please review
